lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -280,6 +285,20 @@ class PatternStopCard extends Component<Props> {
   }
 }
 
+/**
+ * Whether a pattern halt is a flex location/location group.
+ */
+function isFlexLocationOrLocationGroup (patternStop: PatternHalt) {
+  return !!patternHaltIsLocation(patternStop) || !!patternHaltIsLocationGroup(patternStop)
+}
```

Comment:
Could this entire method not be replaced with a negative check for patternStop? I feel this is cleaner than adding an entirely new method

Reply:
Good find, this cleans up more code!